modules/statuses.js

```diff
@@ -0,0 +1,9 @@
+// TODO: Think this is an efficient way of doing it but could be improved
```

Comment:
Theres definitely potential in this :) like the idea of an abstracted status handler